MyGet.ps1

```diff
@@ -0,0 +1,43 @@
+# Performs the build for MyGet and uploads the generated NuGet packages to
+# the "google-dotnet" feed.
+#
+# MyGet's build syncs the repo and subrepos, to build manually be sure to:
+# git submodule update --init ClientGenerator
+
+function PrintHeader($message) {
+    Write-Host "****************************************"
+    Write-Host $message
+    Write-Host "****************************************"
+}
+
+Write-Host ".NET Client Libraries, build shard $Env:BuildShard"
+
+# TODO(chrsmith): Document why we require VS v14 to build.
+[Environment]::SetEnvironmentVariable(
+    "VisualStudioVersion",
+    "14.0",
+    "User")
+
+$msbuild14 = "${Env:ProgramFiles(x86)}\MSBuild\14.0\Bin\MSBuild.exe"
+
+# Build the Support libraries.
+PrintHeader("Building SupportLibraries.proj")
+& $msbuild14 /m /nr:false SupportLibraries.proj
+
+# TODO(chrsmith): Run unit tests as applicable.
+# At HEAD we have some test failures due to portable libraries and the
+# venerable System.Net.Http.Primitives assembly.
+
+# Install dependent Python libraries.
+PrintHeader("Installing Python dependencies")
```

Comment:
Done. `Print-Header` might be even better then? I'm not sure the convention for local functions vs. proper cmdlets...